site/_resources/envoy.md

```diff
@@ -34,13 +34,12 @@ Contour requires the following extensions.
 If you are using the image recommended in our [example deployment][4] no action is required.
 If you are providing your own Envoy it must be compiled with the following extensions:
 
-- `access_loggers`: `envoy.file_access_log`,`envoy.http_grpc_access_log`,`envoy.tcp_grpc_access_log`
-- `filters.http`: `envoy.buffer`,`envoy.cors`,`envoy.csrf`,`envoy.ext_authz`,`envoy.fault`,`envoy.filters.http.adaptive_concurrency`,`envoy.filters.http.dynamic_forward_proxy`,`envoy.filters.http.grpc_http1_reverse_bridge`,`envoy.filters.http.grpc_stats`,`envoy.filters.http.header_to_metadata`,`envoy.filters.http.jwt_authn`,`envoy.filters.http.original_src`,`envoy.filters.http.rbac`,`envoy.filters.http.tap`,`envoy.grpc_http1_bridge`,`envoy.grpc_json_transcoder`,`envoy.grpc_web`,`envoy.gzip`,`envoy.health_check`,`envoy.ip_tagging`,`envoy.rate_limit`,`envoy.router`,`envoy.squash`
+- `access_loggers`: `envoy.file`,`envoy.http_grpc`,`envoy.tcp_grpc`
```

Comment:
I think these have been renamed to

“envoy.access_loggers.file”

“envoy.access_loggers.http_grpc”

“envoy.access_loggers.tcp_grpc”

https://www.envoyproxy.io/docs/envoy/latest/api-v2/config/filter/accesslog/v2/accesslog.proto

Reply:
There were some others that I updated. Do we need to version this extension list?